src/test/java/seedu/bigpp/command/buildercommand/BuilderListComponentCommandTest.java

```
@@ -140,7 +141,7 @@ public class BuilderListComponentCommandTest {
             "SOCKET: AM5\n" +
             "================\n";
     @Test
-    public void executeCommand_listComponentCommand_success() {
+    public void executeCommand_listComponentCommand_success() throws PPException {
```

Need to throw the specific exception over here for more clarity